### PR TITLE
enable cd for pipeline-input-notification

### PR DESCRIPTION
**Pull request by yuezhuangshi**
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: .github/workflows/cd.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
+
+name: cd
+on:
+  workflow_dispatch:
+#  check_run:
+#    types:
+#      - completed
```
Comment (NotMyFault) [inline .github/workflows/cd.yaml:8]:
```suggestion
  check_run:
    types:
      - completed
```

**Review by NotMyFault — CHANGES_REQUESTED** [review]
This is needed in order to make CD run.

File: .github/workflows/cd.yaml (hunk 1 above)
Comment (yuezhuangshi) [reply .github/workflows/cd.yaml:8]: I read that it is mentioned in the [document](https://www.jenkins.io/doc/developer/publishing/releasing-cd/#releasing), if you want to trigger a deployment explicitly, you need to comment out the `check_run` section. Am I misunderstanding here?

Comment (NotMyFault) [reply .github/workflows/cd.yaml:8]: > Am I misunderstanding here?

Yes. You can trigger a release still, alongside automatic releases. Commenting out is not needed. If you don't want automatic releases, CD is not the right way to go. You should stick to the maven release plugin then.

Comment (yuezhuangshi) [reply .github/workflows/cd.yaml]: Got it, thanks for your guidance

**Pull request merged**